webapp/src/main/java/com/box/l10n/mojito/service/tm/TMService.java

```diff
@@ -709,7 +710,11 @@ public String generateLocalized(
 
         driver.addStep(new RawDocumentToFilterEventsStep());
         driver.addStep(new CheckForDoNotTranslateStep());
-        driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
+        if (bcp47Tag.equals("en-x-psaccent")) {
```

Comment:
invert the test `"en-x-psaccent".equals(bcp47Tag)`, it is better code to avoid NPE
introduce a constant for `"en-x-psaccent"`